www/src/components/landingPage/button.astro

```diff
@@ -0,0 +1,39 @@
+---
+export interface Props {
+  href?: string;
+  openInNewTab?: boolean;
+  rounded?: "full" | "md" | "none";
+  variant?: "primary" | "secondary";
+  specialHover?: boolean;
+  className?: string;
+}
+
+const {
+  href = "",
+  rounded = "md",
+  variant = "primary",
+  openInNewTab = false,
+  className = "",
+  specialHover = false,
+} = Astro.props;
+
+const roundedClass =
+  rounded === "full" ? "rounded-full" : rounded === "md" ? "rounded-md" : "";
+const variantClass =
+  variant === "primary"
+    ? "bg-t3-purple-100 hover:bg-t3-purple-200 text-slate-800"
+    : "bg-white/10 hover:bg-white/20 text-white";
+
+const specialHoverClass = specialHover
+  ? "hover:shadow hover:shadow-[#300171] duration-500"
+  : "";
+---
+
+<a
+  href={href}
+  target={openInNewTab ? "_blank" : "_self"}
+  rel="noopener noreferrer"
+  class={`${specialHoverClass} ${roundedClass} ${variantClass} ${className} inline-flex items-center px-3 md:px-5 py-2 md:py-3 text-sm md:text-base font-semibold cursor-pointer hover:no-underline transition-all`}
```

Comment:
I think the small buttons could have a bit more padding. Currently they are 36px high on phone screens

https://www.w3.org/WAI/WCAG21/Understanding/target-size.html
recommends min. 44x44

<img width="1552" alt="image" src="https://user-images.githubusercontent.com/8353666/193416462-840db954-8fd5-46c6-94ee-9102b49f74c1.png">